docs: add Events section to README (#2655) (#2773)
camel-ai/camel: README.md

<br>

## 🗓️ Events

We are actively involved in community events including:

- 🎙️ **Community Meetings** — Weekly virtual syncs with the CAMEL team
- 🏆 **Competitions** — Hackathons, Bounty Tasks and coding challenges hosted by CAMEL
- 🤝 **Volunteer Activities** — Contributions, documentation drives, and mentorship
- 🌍 **Ambassador Programs** — Represent CAMEL in your university or local tech groups 

> > Want to host or participate in a CAMEL event? Join the [Discussions](https://github.com/camel-ai/camel/discussions) or want to be part of [Ambassador Program](https://www.camel-ai.org/ambassador).

Reviewer: ```suggestion
> Want to host or participate in a CAMEL event? Join our [Discord](https://discord.com/invite/CNcNpquyDc) or want to be part of [Ambassador Program](https://www.camel-ai.org/ambassador).
```
